cmd/katapult/client_test.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/krystal/go-katapult"
+
+	"github.com/krystal/katapult-cli/config"
+)
+
+func TestNewClient(t *testing.T) {
```

Comment:
Nitpick: I believe this test should be called `Test_newClient` as the function it's testing is called `newClient`.